### PR TITLE
GH-43 Add limit on the amount of money sent in one transaction.

### PR DESCRIPTION
**Pull request by VerniQ**

### CODE REVIEW

**Review by CitralFlo — APPROVED** [review]
Overall good job

File: eternaleconomy-core/src/main/java/com/eternalcode/economy/command/player/MoneyTransferCommand.java [hunk 1]
```diff
@@ -45,6 +49,16 @@ void execute(@Context Account payer, @Arg @NotSender Account receiver, @Arg @Pos
             return;
         }
 
+        if(amount.compareTo(this.pluginConfig.transactionLimit) > 0) {
```
Comment (Rollczi) [inline eternaleconomy-core/src/main/java/com/eternalcode/economy/command/player/MoneyTransferCommand.java:52]: Space

**Review by Rollczi — APPROVED** [review]

File: eternaleconomy-core/src/main/java/com/eternalcode/economy/command/player/MoneyTransferCommand.java (hunk 1 above)
Comment (Rollczi) [inline eternaleconomy-core/src/main/java/com/eternalcode/economy/command/player/MoneyTransferCommand.java:52]:
```suggestion
        if (amount.compareTo(this.pluginConfig.transactionLimit) > 0) {
```

File: eternaleconomy-core/src/main/java/com/eternalcode/economy/config/implementation/messages/MessagesPlayerSubSection.java [hunk 2]
```diff
@@ -27,4 +27,6 @@ public class MessagesPlayerSubSection extends OkaeriConfig {
     public Notice transferReceived = Notice.chat("<b><gradient:#00FFA2:#34AE00>ECONOMY</gradient></b> "
         + "<dark_gray>➤</dark_gray> <white>Received <gradient:#00FFA2:#34AE00>{AMOUNT}</gradient> from "
         + "<gradient:#00FFA2:#34AE00>{PLAYER}</gradient>.</white>");
+    public Notice transferLimit = Notice.chat("<b><gradient:#00FFA2:#34AE00>ECONOMY</gradient></b> <dark_gray>➤</dark_gray>"
+        + " <white>Transaction limit is <gradient:#00FFA2:#34AE00>{LIMIT}</gradient>.</white>");
```
[the remark this answers is not in the document]
Comment (vLuckyyy) [reply eternaleconomy-core/src/main/java/com/eternalcode/economy/config/implementation/messages/MessagesPlayerSubSection.java]: it won't match the plugin's color scheme, let's leave it as it is

**Review by vLuckyyy — APPROVED** [review]

**Pull request merged**